### PR TITLE
Added rodadahacker.org - #28

### PR DESCRIPTION
**Pull request by hi-hi-ray**
Resolves Issue #28

This pull request has the finality to add RodAda Hacker to the list in the part of Meetups/Events in subsection General.

### CODE REVIEW

**Comment by hi-hi-ray** [comment]
I guess it's having conflicts, How can i Resolve?

File: README.md [hunk 1]
```diff
@@ -84,16 +84,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     League of Women Coders (formerly Ladies Who Code, est. June 2011) is a meetup that focuses on coding, hacking, and learning collaboratively. The group meets up at least once a month in New York City and Boston. The meetups focus on talks, technical questions and ideation, and occasionally feature female speakers.
 
+  - [RodAda Hacker](http://rodadahacker.org/)  - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+
+    RodAda Hacker is a brazillian network focused on stimulating the appropriation of new technologies by girls and women, which is based on the realization of collaborative workshops specially designed for the female audience. The name of the project is a tribute to Ada Lovelace.
```
Comment (cristianoliveira) [inline README.md]: Although this is a literal translation of their description, to be honest, I think even in Portuguese I didn't understand what they do. I think this part "stimulating the appropriation of new technologies" is quite confuse to me (even in PT)

I found another description in their site that sounds more natural to me:
"A RodAda Hacker é uma rede que se baseia em oficinas de programação especialmente desenhadas para meninas e mulheres. Um encontro para quem quer imaginar e construir projetos incríveis e recriar as tecnologias da rede."

How about something like this:
"The RodAda Hacker is a Brazilian network that provides collaborative workshops of programming specially designed for girls and women. It is a space for who wants to dream, build awesome projects and rethink the technology. The name of the project is a tribute to Ada Lovelace."

Comment (hi-hi-ray) [reply README.md]: It's a great suggestion, i'll change right now.

Comment (cristianoliveira) [reply README.md]: 😄

**Comment by cristianoliveira** [comment]
Looks good to me. Let's wait for the @harsh183 review and we are good to go. 
Obrigado!

File: README.md [hunk 2]
```diff
@@ -84,16 +84,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     League of Women Coders (formerly Ladies Who Code, est. June 2011) is a meetup that focuses on coding, hacking, and learning collaboratively. The group meets up at least once a month in New York City and Boston. The meetups focus on talks, technical questions and ideation, and occasionally feature female speakers.
 
+  - [RodAda Hacker](http://rodadahacker.org/)  - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+
+    The RodAda Hacker is a Brazilian network that provides collaborative workshops of programming specially designed for girls and women. It is a space for who wants to dream, build awesome projects and rethink the technology. The name of the project is a tribute to Ada Lovelace.
```
Comment (harsh183) [inline README.md]: * 'It is a space for who wants to dream, build awesome ' can be 'It is a space for those who want to dream, build awesome ...'

* 'The name of the project is a tribute to Ada Lovelace.' this does not add any special information, and it quite obvious from the name (and every third item in the last is named after her)

Comment (hi-hi-ray) [reply README.md]: I'll change it. =]

File: README.md [hunk 3]
```diff
@@ -84,16 +84,20 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 
     League of Women Coders (formerly Ladies Who Code, est. June 2011) is a meetup that focuses on coding, hacking, and learning collaboratively. The group meets up at least once a month in New York City and Boston. The meetups focus on talks, technical questions and ideation, and occasionally feature female speakers.
 
+  - [RodAda Hacker](http://rodadahacker.org/)  - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+
+    The RodAda Hacker is a Brazilian network that provides collaborative workshops of programming specially designed for girls and women. It is a space for those who want to dream, build awesome projects and rethink the technology.
```
Comment (harsh183) [inline README.md]: Rethink the technology? 

That comes off as a bit add, rethink what technology, or rethink technology in general. The rest of it looks quite great.

Comment (hi-hi-ray) [reply README.md]: I can put "Rethink about technology" or if you have any suggestions, that will be great!!

Comment (harsh183) [reply README.md]: I think 'Rethink about existing technology' might work for us. That said, if you want, you can drop the section entirely if that's not a big activity there and just marketing text. I'll leave it up to you.

Make a comment here once you are done, and I'll work towards solving the merge conflicts.

Comment (hi-hi-ray) [reply README.md]: I'll change. =D 

Sometimes i'm not the great writer so, I guess any tip that i get to make e better text, I embrace it.

Comment (cristianoliveira) [reply README.md]: Hey @hi-hi-ray no worries at all. For a first time contribution, It looks great already 😄

**Comment by hi-hi-ray** [comment]
Can i close the PR?

**Comment by harsh183** [comment]
Okay looks awesome! Thanks for making all those changes! Looking forward to more contributions @hi-hi-ray! 🍰

**Pull request closed without merging**